cypress/e2e/shared/flows.test.ts

```diff
@@ -369,6 +369,6 @@ describe('Flows', () => {
     cy.getByTestID('resource-editable-name').click()
 
     // visualizations should exist
-    cy.getByTestID('giraffe-inner-plot').should('be.visible')
+    cy.getByTestID('giraffe-inner-plot').should('exist')
```

Comment:
They should exist.

Reply:
...but not be visible?

Reply:
I'm wondering why we need to make this change. Did the visualization just get pushed further down the screen so we need to scrollIntoView?

Reply:
Just used the scroll into view and it worked! not sure why it didn't previously!